public/server/db/faq.json

```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "Frequently Asked Questions"
```

Comment:
Unfortunately, we can't add formal definitions like these to the records. Only slang.

Reply:
Okay, I removed it.